README.md

```diff
@@ -27,21 +24,41 @@ hadoop version
 ```
 spark-shell --version
 ```
+---
```

Comment:
why not use the original way "* [Build Bigbench](docs/Bigbench-build.md)"? If Bigbench supports more building ways, setting a separated file to describe how to build Bigbench will be a better way.

Reply:
Logically, The Section "Build Bigbench" should be included in Section "Getting Started", and I didn't delete the docs directory, I just commented it out.